### ISSUE TITLE
Temporary failure resolving 'ports.ubuntu.com'

### ISSUE THREAD

**Issue by danylevskyi** [stephendade/Rpanion-server#119]
**Raspberry Pi 4 Model B**
**Ubuntu Server 20.04**: https://cdimage.ubuntu.com/releases/focal/release/ubuntu-20.04.5-preinstalled-server-armhf+raspi.img.xz

```sh
Err:1 http://ports.ubuntu.com/ubuntu-ports focal-updates/universe armhf wireguard-tools armhf 1.0.20200513-1~20.04.2
  Temporary failure resolving 'ports.ubuntu.com'
Err:2 http://ports.ubuntu.com/ubuntu-ports focal-updates/universe armhf wireguard all 1.0.20200513-1~20.04.2
  Temporary failure resolving 'ports.ubuntu.com'
Err:3 http://ports.ubuntu.com/ubuntu-ports focal/universe armhf resolvconf all 1.82
  Temporary failure resolving 'ports.ubuntu.com'
E: Failed to fetch http://ports.ubuntu.com/ubuntu-ports/pool/universe/w/wireguard/wireguard-tools_1.0.20200513-1~20.04.2_armhf.deb  Temporary failure resolving 'ports.ubuntu.com'
E: Failed to fetch http://ports.ubuntu.com/ubuntu-ports/pool/universe/w/wireguard/wireguard_1.0.20200513-1~20.04.2_all.deb  Temporary failure resolving 'ports.ubuntu.com'
E: Failed to fetch http://ports.ubuntu.com/ubuntu-ports/pool/universe/r/resolvconf/resolvconf_1.82_all.deb  Temporary failure resolving 'ports.ubuntu.com'
E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
```

**Comment by stephendade** [comment]
That error message occurs if there are network connection issues to the Ubuntu apt repository.

You'll need to run the script again

**Comment by stephendade** [comment]
Fixed in #132
